### PR TITLE
Removed not used infrastructures

### PR DESCRIPTION
**Pull request by marcocast**
Removed not used infrastructures and enforced implementation of getDescription method in the abstract InfrastructureManager. If getDescription is not implemented a NullPointerException is thrown at run time. 
The Winhpc infrastructure has been moved to the new and separate nodesource-addons project

### CODE REVIEW

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java [hunk 1]
```diff
@@ -57,662 +57,707 @@
...
+				logger.error("Cannot delete job " + jobID + ". " + deleteCmd + " command timed out.");
+				throw new RMException("Cannot delete job " + jobID + ". " + deleteCmd + " command timed out.");
+			}
+		}
+	}
+
+	/**
+	 * @return short description of the IM
+	 */
+	public String getDescription() {
```
Comment (sonarqube-activeeon) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java]: MAJOR Add the "@Override" annotation above this method signature [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/ow2-proactive/scheduling/blob/04473486f9f366a74c3457633f91e5136b6ebbbe/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)

**Comment by marcocast** [comment]
2 reviews please?

**Comment by tobwiens** [comment]
Okay, you could add the @Override as sonar suggest and convert to white spaces.

**Comment by marcocast** [comment]
@tobwiens  Done.

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)
* MAJOR [Reduce this anonymous class number of lines from 22 to at most 20, or make it a named class.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java#L258) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)

**Comment by marcocast** [comment]
Retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)
* MAJOR [Reduce this anonymous class number of lines from 22 to at most 20, or make it a named class.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java#L258) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)

**Comment by marcocast** [comment]
Retest this please

**Comment by marcocast** [comment]
Retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)
* MAJOR [Reduce this anonymous class number of lines from 22 to at most 20, or make it a named class.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java#L258) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)

**Comment by marcocast** [comment]
Retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)
* MAJOR [Reduce this anonymous class number of lines from 22 to at most 20, or make it a named class.](https://github.com/ow2-proactive/scheduling/blob/10247a21d11ea86b4b38276cbb016923ae52d06a/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java#L258) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)

**Pull request merged**